patches/chrome-common-importer-profile_import.mojom.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/common/importer/profile_import.mojom b/chrome/common/importer/profile_import.mojom
+index 6ea9e8f6935382908a46a53f955251fd7016f546..ff1e641e6508e2e07c71a479b5e6f201531b6771 100644
+--- a/chrome/common/importer/profile_import.mojom
++++ b/chrome/common/importer/profile_import.mojom
+@@ -62,6 +62,7 @@ interface ProfileImportObserver {
+   OnAutofillFormDataImportStart(uint32 total_autofill_form_data_entry_count);
+   OnAutofillFormDataImportGroup(
+       array<ImporterAutofillFormDataEntry> autofill_form_data_entry_group);
++  OnCreditCardImportReady(mojo_base.mojom.String16 name_on_card, mojo_base.mojom.String16 expiration_month, mojo_base.mojom.String16 expiration_year, mojo_base.mojom.String16 decrypted_card_number, string origin);
```

Comment:
classes can implement more than one mojo interface so can we just pull this entire thing out into our own code that only the brave importer uses?

Reply:
Ok, it's good to introduce new mojo interface.
While I'm working on this, I hesitated to add new one because it will add more new patch files :)

Reply:
Tried to use separate mojo interface (`BraveProfileImportObserver`) for our new apis.
However, it will introduce a lot of patches because our observer instance should be passed via upstream `ProfileImport` interface and it means each side's bridge ctor parameter changes to pass it.
So, how about modifying this file directly? @bridiver

Reply:
I'll build this branch and look at it because the point was to eliminate the patches by moving the methods into a separate interface